compiler/rustc_const_eval/src/interpret/eval_context.rs

```diff
@@ -825,132 +482,6 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
         self.size_and_align_of(&mplace.meta(), &mplace.layout)
     }
 
-    #[instrument(skip(self, body, return_place, return_to_block), level = "debug")]
-    pub fn push_stack_frame(
-        &mut self,
-        instance: ty::Instance<'tcx>,
-        body: &'tcx mir::Body<'tcx>,
-        return_place: &MPlaceTy<'tcx, M::Provenance>,
-        return_to_block: StackPopCleanup,
-    ) -> InterpResult<'tcx> {
-        trace!("body: {:#?}", body);
-
-        // First push a stack frame so we have access to the local args
-        self.push_new_stack_frame(instance, body, return_to_block, return_place.clone())?;
-
-        self.after_stack_frame_push(instance, body)?;
```

Comment:
@WaffleLapkin `push_new_stack_frame` and `after_stack_frame_push` both had only one call site, right here. Not sure why you split them up into separate functions in https://github.com/rust-lang/rust/pull/113128; I've merged them back together here.

Reply:
I'm pretty sure it's an artifact of when I was trying some other design to what I ended up doing.